crates/llm-base/src/vocabulary.rs

```diff
@@ -1,6 +1,7 @@
 use std::{collections::HashMap, error::Error, fmt::Display, str::FromStr};
 
 use crate::InferenceError;
+use std::str;
```

Comment:
Is this needed?

Reply:
no sorry, I forgot to remove it, i think i added it when i was trying to make the tokenizer work

Reply:
Honestly I'm surprised Clippy didn't catch it 🤔